concrete/elements/page_controls_footer.php

```diff
@@ -94,12 +94,13 @@
                         </li>
                         <?php
                     } elseif ($permissions->canEditPageContents()) {
+                        $request = Request::createFromGlobals();
```

Comment:
Better to use the request singleton here (that is, `$request = app(Request::class);`)